### PR TITLE
Changed prs_per_repo.rb to use Sprints instead of Milestones

### PR DESCRIPTION
**Pull request by chessbyte**
Followed style from [merged_prs_per_sprint.rb](https://github.com/ManageIQ/sprint_statistics/blob/master/merged_prs_per_sprint.rb)

### CODE REVIEW

File: sprint_statistics.rb [hunk 1]
```diff
@@ -2,33 +2,12 @@
 require 'active_support/core_ext'
 
 class SprintStatistics
-  def initialize(access_token, milestone_string = nil)
```
Comment (bdunne) [inline sprint_statistics.rb:5]: Other scripts are still using this `milestone_string`

Comment (chessbyte) [reply sprint_statistics.rb:5]: Should I update the callers in this PR or in followup PRs?  I am not sure if they even work.

**Comment by chessbyte** [comment]
Setting to WIP as the computed numbers seem wrong.

File: sprint_statistics.rb (hunk 1 above)
Comment (bdunne) [reply sprint_statistics.rb:5]: Changes in this file aren't necessary for the rest of the PR, right?

Comment (chessbyte) [reply sprint_statistics.rb:5]: No.  Will remove the unrelated changes to remove milestone.

**Comment by chessbyte** [comment]
Think I got things fixed now.  Removing WIP.

**Pull request merged**